pycm/pycm_test.py

```diff
@@ -18,5 +18,14 @@
 >>> cp = Compare({"cm1":cm,"cm2":cm2})
 >>> cp
 pycm.Compare(classes: ['Class1', 'Class2', 'Class3'])
+>>> cv = Curve(actual_vector = [1, 1, 2, 2], probs = [[0.1, 0.9], [0.4, 0.6], [0.35, 0.65], [0.8, 0.2]], classes=[2,1])
```

Comment:
`actual_vector=[1, 1, 2, 2], probs=[[0.1, 0.9], [0.4, 0.6], [0.35, 0.65], [0.8, 0.2]], classes=[2, 1]`

Reply:
fixed in https://github.com/sepandhaghighi/pycm/pull/457/commits/b49a1624972ceb4b627263ee299fb25350e8319d